### PR TITLE
Added this-week-in-os-17

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Created file
- Added og

### CODE REVIEW

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+- [mainmatter/eurorust.eu] [#179](https://github.com/mainmatter/eurorust.eu/pull/179) fix(frontpage): increate max-width for premier logos ([@oscard0m])
+- [mainmatter/eurorust.eu] [#178](https://github.com/mainmatter/eurorust.eu/pull/178) fix(frontpage): replace ferrous systems logo with ferrocene logo ([@oscard0m])
+- [mainmatter/eurorust.eu] [#176](https://github.com/mainmatter/eurorust.eu/pull/176) feat(frontpage): add OxidOS as supporter sponsor ([@oscard0m])
+- [mainmatter/eurorust.eu] [#174](https://github.com/mainmatter/eurorust.eu/pull/174) fix(frontpage): increase size for supporter logos ([@oscard0m])
+- [mainmatter/eurorust.eu] [#173](https://github.com/mainmatter/eurorust.eu/pull/173) fix(frontpage): improve CSS to keep aspect ratio and size between different sponsorships ([@oscard0m])
+- [mainmatter/eurorust.eu] [#172](https://github.com/mainmatter/eurorust.eu/pull/172) feat(frontpage): replace 'ada core' with ferrous systems as premier sponsor ([@oscard0m])
+- [mainmatter/eurorust.eu] [#170](https://github.com/mainmatter/eurorust.eu/pull/170) feat(frontpage): add Grafbase as supporter ([@oscard0m])
```
Comment (oscard0m) [inline src/posts/2022-10-14-this-week-in-os-17.md]: Ignore `mainmatter/eurorust.eu`

```suggestion
```

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+- [mainmatter/gravity] [#951](https://github.com/mainmatter/gravity/pull/951) fix(@gravity/dashboard): use environment variables for GitHub App secrets ([@oscard0m])
+- [mainmatter/gravity] [#950](https://github.com/mainmatter/gravity/pull/950) docs(readme): add documentation about GitHub Login in our Gravity Dashboard ([@oscard0m])
+- [mainmatter/gravity] [#943](https://github.com/mainmatter/gravity/pull/943) [spike] persist user session in dashboard ([@oscard0m])
+- [mainmatter/gravity] [#955](https://github.com/mainmatter/gravity/pull/955) Integrate login logic with routes ([@inesilva])
+- [mainmatter/gravity] [#949](https://github.com/mainmatter/gravity/pull/949) 404 screen ([@inesilva])
+- [mainmatter/gravity] [#942](https://github.com/mainmatter/gravity/pull/942) Create Accounts screen ([@inesilva])
+- [mainmatter/gravity] [#938](https://github.com/mainmatter/gravity/pull/938) Clean up code ([@inesilva])
+- [mainmatter/gravity] [#932](https://github.com/mainmatter/gravity/pull/932) Repositories screen ([@inesilva])
```
Comment (oscard0m) [inline src/posts/2022-10-14-this-week-in-os-17.md]: ignore `mainmatter/gravity`

```suggestion
```

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+[anuraghazra/github-readme-stats]: https://github.com/anuraghazra/github-readme-stats
+[chrislgarry/Apollo-11]: https://github.com/chrislgarry/Apollo-11
+[empress/empress-blog]: https://github.com/empress/empress-blog
+[git/git]: https://github.com/git/git
+[hoppscotch/hoppscotch]: https://github.com/hoppscotch/hoppscotch
+[huggingface/transformers]: https://github.com/huggingface/transformers
+[jomjol/AI-on-the-edge-device]: https://github.com/jomjol/AI-on-the-edge-device
+[mainmatter/ember2-x-codemods]: https://github.com/mainmatter/ember2-x-codemods
+[mainmatter/eurorust.eu]: https://github.com/mainmatter/eurorust.eu
+[mainmatter/gravity]: https://github.com/mainmatter/gravity
```
Comment (oscard0m) [inline src/posts/2022-10-14-this-week-in-os-17.md]:
```suggestion
```

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+## JavaScript
+
+- [strapi/strapi] [#14516](https://github.com/strapi/strapi/pull/14516) ci(checks.yml): update actions/checkout to v3 ([@oscard0m])
+
+## Unknown
+
+- [AeroRust/nmea] [#59](https://github.com/AeroRust/nmea/pull/59) Implement `PGRMZ` support ([@Turbo87])
+- [AeroRust/nmea] [#57](https://github.com/AeroRust/nmea/pull/57) Update/Fix `RMC` docs ([@Turbo87])
+- [Swatinem/rust-cache] [#83](https://github.com/Swatinem/rust-cache/pull/83) Update `@actions/cors` to v1.10.0 ([@Turbo87])
```
Comment (Turbo87) [inline src/posts/2022-10-14-this-week-in-os-17.md]: This and the two above are "Rust"

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+- [hoppscotch/hoppscotch] [#2725](https://github.com/hoppscotch/hoppscotch/pull/2725) ci(deploy-netlify): upgrade actions/checkout to v3 ([@oscard0m])
+- [hoppscotch/hoppscotch] [#2724](https://github.com/hoppscotch/hoppscotch/pull/2724) ci(codeql-analysis): upgrade actions/checkout to v3 ([@oscard0m])
+- [hoppscotch/hoppscotch] [#2723](https://github.com/hoppscotch/hoppscotch/pull/2723) ci(deploy-prod): upgrade actions/checkout to v3 ([@oscard0m])
+- [hoppscotch/hoppscotch] [#2722](https://github.com/hoppscotch/hoppscotch/pull/2722) ci(deploy-staging-netlify): upgrade actions/checkout to v3 ([@oscard0m])
+- [hoppscotch/hoppscotch] [#2721](https://github.com/hoppscotch/hoppscotch/pull/2721) ci(publish-docker): upgrade actions/checkout to v3 ([@oscard0m])
+- [huggingface/transformers] [#19281](https://github.com/huggingface/transformers/pull/19281) ci(stale.yml): upgrade actions/setup-python to v4 ([@oscard0m])
+- [huggingface/transformers] [#19280](https://github.com/huggingface/transformers/pull/19280) ci(stale.yml): update actions/checkout to v3 ([@oscard0m])
```
Comment (oscard0m) [inline src/posts/2022-10-14-this-week-in-os-17.md]: irrelevant PRs

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+- [jomjol/AI-on-the-edge-device] [#1158](https://github.com/jomjol/AI-on-the-edge-device/pull/1158) FlowPostProcessing::GetJSON: Add `pre` field to the `/json` response ([@Turbo87])
```
Comment (Turbo87) [inline src/posts/2022-10-14-this-week-in-os-17.md]: "Embedded" I guess

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+- [nestjs/nest] [#10335](https://github.com/nestjs/nest/pull/10335) ci(codeql-analysis): upgrade actions/checkout to v3 ([@oscard0m])
+- [ohmyzsh/ohmyzsh] [#11209](https://github.com/ohmyzsh/ohmyzsh/pull/11209) ci(workflows): update actions/checkout to v3 ([@oscard0m])
+- [scrapy/scrapy] [#5652](https://github.com/scrapy/scrapy/pull/5652) ci(test-macos): upgrade actions/setup-python to v4 ([@oscard0m])
+- [scrapy/scrapy] [#5651](https://github.com/scrapy/scrapy/pull/5651) ci(test-macos): upgrade actions/checkout to v3 ([@oscard0m])
+- [scrapy/scrapy] [#5650](https://github.com/scrapy/scrapy/pull/5650) ci(tests-windows): upgrade actions/checkout to v3 ([@oscard0m])
+- [scrapy/scrapy] [#5649](https://github.com/scrapy/scrapy/pull/5649) ci(tests-windows): upgrade actions/setup-python to v4 ([@oscard0m])
+- [sharkdp/bat] [#2341](https://github.com/sharkdp/bat/pull/2341) ci(CICD.yml): upgrade actions/checkout to v3 ([@oscard0m])
+- [tldr-pages/tldr] [#8615](https://github.com/tldr-pages/tldr/pull/8615) stale.yml: upgrade actions/stale to v6 ([@oscard0m])
+- [tldr-pages/tldr] [#8614](https://github.com/tldr-pages/tldr/pull/8614) ci: upgrade actions/setup-python to v4 ([@oscard0m])
```
Comment (oscard0m) [inline src/posts/2022-10-14-this-week-in-os-17.md]: irrelevant PRs
```suggestion
```

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+- [withastro/docs] [#1662](https://github.com/withastro/docs/pull/1662) i18n(es): Update sharing-state.md translation ([@oscard0m])
```
Comment (oscard0m) [inline src/posts/2022-10-14-this-week-in-os-17.md]: category: Astro

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+[huggingface/transformers]: https://github.com/huggingface/transformers
+[jomjol/AI-on-the-edge-device]: https://github.com/jomjol/AI-on-the-edge-device
+[mainmatter/ember2-x-codemods]: https://github.com/mainmatter/ember2-x-codemods
+[mainmatter/eurorust.eu]: https://github.com/mainmatter/eurorust.eu
+[mainmatter/gravity]: https://github.com/mainmatter/gravity
+[mansona/lint-to-the-future]: https://github.com/mansona/lint-to-the-future
+[microsoft/Web-Dev-For-Beginners]: https://github.com/microsoft/Web-Dev-For-Beginners
+[narze/profile-updater]: https://github.com/narze/profile-updater
+[nestjs/nest]: https://github.com/nestjs/nest
+[ohmyzsh/ohmyzsh]: https://github.com/ohmyzsh/ohmyzsh
```
Comment (oscard0m) [inline src/posts/2022-10-14-this-week-in-os-17.md]:
```suggestion
```

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+[rust-lang/crates.io]: https://github.com/rust-lang/crates.io
+[scrapy/scrapy]: https://github.com/scrapy/scrapy
+[sharkdp/bat]: https://github.com/sharkdp/bat
+[strapi/strapi]: https://github.com/strapi/strapi
+[tldr-pages/tldr]: https://github.com/tldr-pages/tldr
```
Comment (oscard0m) [inline src/posts/2022-10-14-this-week-in-os-17.md]:
```suggestion
```

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+
+- [empress/empress-blog] [#163](https://github.com/empress/empress-blog/pull/163) update ember-scroll to 1.0.3 ([@nickschot])
+
+## Lint to the future
+
+- [mansona/lint-to-the-future] [#28](https://github.com/mansona/lint-to-the-future/pull/28) Move from minimist to commander ([@Mikek2252])
+
+## JavaScript
+
+- [strapi/strapi] [#14516](https://github.com/strapi/strapi/pull/14516) ci(checks.yml): update actions/checkout to v3 ([@oscard0m])
```
Comment (oscard0m) [inline src/posts/2022-10-14-this-week-in-os-17.md]:
```suggestion
```

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 12]
```diff
@@ -0,0 +1,139 @@
...
+[@Mikek2252]: https://github.com/Mikek2252
+[@Turbo87]: https://github.com/Turbo87
+[@inesilva]: https://github.com/inesilva
+[@marcoow]: https://github.com/marcoow
+[@nickschot]: https://github.com/nickschot
+[@oscard0m]: https://github.com/oscard0m
+[AeroRust/nmea]: https://github.com/AeroRust/nmea
+[Swatinem/rust-cache]: https://github.com/Swatinem/rust-cache
+[anuraghazra/github-readme-stats]: https://github.com/anuraghazra/github-readme-stats
+[chrislgarry/Apollo-11]: https://github.com/chrislgarry/Apollo-11
```
Comment (oscard0m) [inline src/posts/2022-10-14-this-week-in-os-17.md]:
```suggestion
```

File: src/posts/2022-10-14-this-week-in-os-17.md [hunk 13]
```diff
@@ -0,0 +1,139 @@
...
+[empress/empress-blog]: https://github.com/empress/empress-blog
+[git/git]: https://github.com/git/git
+[hoppscotch/hoppscotch]: https://github.com/hoppscotch/hoppscotch
+[huggingface/transformers]: https://github.com/huggingface/transformers
```
Comment (oscard0m) [inline src/posts/2022-10-14-this-week-in-os-17.md]:
```suggestion
```

**Review by oscard0m — CHANGES_REQUESTED** [review]

File: src/posts/2022-10-14-this-week-in-os-17.md (hunk 6 above)
Comment (emmasofiah2o) [reply src/posts/2022-10-14-this-week-in-os-17.md]: All of them?

File: src/posts/2022-10-14-this-week-in-os-17.md (hunk 8 above)
Comment (emmasofiah2o) [reply src/posts/2022-10-14-this-week-in-os-17.md]: Just line 109 right?

File: src/posts/2022-10-14-this-week-in-os-17.md (hunk 6 above)
Comment (Turbo87) [reply src/posts/2022-10-14-this-week-in-os-17.md]: no, sorry, just the one from `jomjol/AI-on-the-edge-device` :)

**Review by oscard0m — APPROVED** [review]

**Pull request merged**